package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Did you clear out your package.json?

Reply:
@DaniPad7 , are you working with an empty package.json?

Reply:
@honpanloi @MichaelPearson-gif @DaniPad7 @Sofia-S-S Your package.json is empty?

Reply:
@russchristina did you mean Michael Ball? I'm the Michael that worked on the backend